app/actions/device/index.js

```diff
@@ -1,14 +1,19 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {networkStatusChanged} from 'redux-offline/lib/actions';
```

Comment:
`redux-offline` is only at package-lock.json. Maybe we need to declare it at `package.json`.

Reply:
do we need it? it is a dependency of `mattermost-redux` so I'm 0/5 here

Reply:
I think so, so that we're explicit on which library/version we're using and will not cause conflict in the future if in case we'll add a dependency with redux-offline on it as its dependency.